src/components/Seperator.vue

```diff
@@ -0,0 +1,16 @@
+<template>
```

Comment:
IMHO this "component" is useless

Reply:
@kevinley is right. We could use one of Quasar: https://quasar.dev/vue-components/separator#Usage and customize it.